cypress/integration/admin/loggingInAndOut.js

```diff
@@ -31,6 +31,7 @@ describe("As an admin, I want to log in so that I can access the service.", () =
     ThenISeeAnError();
   });
 
+  // Is this test still relevant?
```

Comment:
Did you answer this question/whats the background to this?

Reply:
Test isn't required anymore, so I've removed it